### PR TITLE
Data sync to custom Salesforce objects

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Added a section on syncing data to custom Salesforce objects using VDM.

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/sources/reverse-etl/features/visual-data-mapper.mdx [hunk 1]
```diff
@@ -108,36 +99,54 @@ When sending events to some objects, there are some required fields that need to
...
 
-RudderStack lets you create a custom destination field and map it to a warehouse column by following the below steps:
+Once you have mapped all the source columns to the destination fields, click on **Next** to complete the destination configuration.
 
-1. Click on the **Map another field** option. 
-2. Type the name of the custom destination field that you want to create, as shown:
+### Syncing data to custom Salesforce objects using VDM
 
-<img src="../../../assets/warehouse-actions-sources/vdm-7.png" alt="Map another field" />
+RudderStack **does not support** creating new custom <Link to="/destinations/streaming-destinations/salesforce/">Salesforce</Link> objects using the above-mentioned steps. However, you can still use the Visual Data Mapping feature sync your data to these objects by following the below steps:
```
Comment (ruchiramoitra) [inline docs/sources/reverse-etl/features/visual-data-mapper.mdx:135]: the above mentioned steps are about creating custom fields not objects. We do not allow creation of custom fields in salesforce.

Comment (ruchiramoitra) [reply docs/sources/reverse-etl/features/visual-data-mapper.mdx:135]: for none custom objects can be created.

Comment (ameypv-rudder) [inline docs/sources/reverse-etl/features/visual-data-mapper.mdx:135]:
```suggestion
RudderStack **does not support** creating new custom fields for the <Link to="/destinations/streaming-destinations/salesforce/">Salesforce</Link> destination using the above-mentioned steps. 

However, you can still use the Visual Data Mapping feature sync your data to the custom Salesforce objects by following the below steps:
```

Comment (ameypv-rudder) [reply docs/sources/reverse-etl/features/visual-data-mapper.mdx]: thanks @ruchiramoitra I've updated the section accordingly.

**Review by ruchiramoitra — APPROVED** [review]

File: docs/sources/reverse-etl/features/visual-data-mapper.mdx [hunk 2]
```diff
@@ -108,36 +99,56 @@ When sending events to some objects, there are some required fields that need to
...
+  <li><a href="https://rudderstack.com/docs/destinations/marketing/klaviyo/">Klaviyo</a></li>
 </ul>
 </div>
 
-RudderStack lets you create a custom destination field and map it to a warehouse column by following the below steps:
+Once you have mapped all the source columns to the destination fields, click on **Next** to complete the destination configuration.
 
-1. Click on the **Map another field** option. 
-2. Type the name of the custom destination field that you want to create, as shown:
+### Syncing data to custom Salesforce objects using VDM
```
Comment (ishachopra0401) [inline docs/sources/reverse-etl/features/visual-data-mapper.mdx:133]:
```suggestion
#### Syncing data to custom Salesforce objects
```

File: docs/sources/reverse-etl/features/visual-data-mapper.mdx [hunk 3]
```diff
@@ -108,36 +99,56 @@ When sending events to some objects, there are some required fields that need to
...
 
-<img src="../../../assets/warehouse-actions-sources/vdm-7.png" alt="Map another field" />
+RudderStack **does not support** creating new custom fields for the <Link to="/destinations/streaming-destinations/salesforce/">Salesforce</Link> destination using the above-mentioned steps. 
 
-3. Finally, click on **Create**.
+However, you can still use the Visual Data Mapping feature sync your data to the custom Salesforce objects by following the below steps:
```
Comment (ishachopra0401) [inline docs/sources/reverse-etl/features/visual-data-mapper.mdx:137]:
```suggestion
However, you can still use the Visual Data Mapping feature to sync your data to the custom Salesforce objects by following the below steps:
```

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**